src/Repl/Repl.tsx

```diff
@@ -311,15 +311,15 @@ const Repl = (props) => {
       let pc = props.programCode;
       // only run this when we have program code (in testing this is not defined)
       if(pc !== undefined) {
-        let pCheck = pc.replaceAll(/(?:--[^\n]*)/g,'').trim();
+        let pCheck = removeComments(pc)
```

Comment:
Pulled this out into a singular function that removes single-line & block comments.